lib/stan.js

```diff
@@ -21,7 +21,7 @@ var util = require('util'),
 /**
  * Constants
  */
-var VERSION = '0.0.3',
+var VERSION = '0.0.4',
```

Comment:
It is published. You can also reference from npm the github repo to get the current master.